includes/class-wc-siftscience-orders.php

```diff
@@ -61,7 +61,7 @@ public function create_row( $column ) {
 		 */
 		private function add_react_app() {
 			$data = array(
-				'imgPath'       => plugins_url( 'images/', dirname( __FILE__ ) ),
+				'imgPath'       => plugin_dir_url( __DIR__ ) . 'dist/images/',
```

Comment:
Ah cool. I was worried that the VueJS code would need updating, but I guess not :-)